modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="cluster-tagging_{context}"]
+= Tagging an existing cluster as owned
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster. These resources include:
+
+* All hosts.
+* All relevant load balancers to be used in the AWS instances.
+* All relevant security groups to be used with the AWS instances.
```

Comment:
I do not think we should mention ELB and SG here, since scale up will use existing ones, which are the same as OCP cluster .